config/services/register-as-childminder.json

```diff
@@ -0,0 +1,145 @@
+{
+  "base_path": "/services/registering-as-childminder-england",
+  "content_id": "4ba1356d-c4c7-4f6b-7777-3b9a52af1ddd",
+  "document_type": "service",
+  "government_document_supertype": "other",
+  "navigation_document_supertype": "guidance",
+  "locale": "en",
+  "user_journey_document_supertype": "service",
+  "title": "Register as a childminder (England)",
+  "publishing_app": "publisher",
+  "rendering_app": "collections",
+  "rendering_type": "accordion",
+  "schema_name": "service",
+  "links": {
+    "ordered_steps": [
+      {
+        "content_id": "6e7c1795-2592-4ada-adc7-2cdc478a607f",
+        "title": "Check if you are eligible",
+        "description": "You will need to fit certain criterea to get on the early years register.",
+        "ordered_tasks": [
+          {
```

Comment:
Perhaps the task itself might be a separate content type that links to the content on GOV.UK?